extension/EdFi.Ods.Extensions.LMSX/LMS-Harmonizer/1090-lms-assignment-submissions.sql

```diff
@@ -73,29 +69,32 @@ BEGIN
 	FROM edfi.StudentSectionAssociation edfisectionassociation
 	INNER JOIN lmsx.Assignment lmsxassignment
 		ON edfisectionassociation.SectionIdentifier = lmsxassignment.SectionIdentifier
+	INNER JOIN lms.Assignment lmsassignment
+		ON lmsassignment.SourceSystemIdentifier = lmsxassignment.AssignmentIdentifier
 	INNER JOIN edfi.Student edfistudent
 		ON edfistudent.StudentUSI = edfisectionassociation.StudentUSI
 	INNER JOIN lms.LMSUser lmsstudent
 		ON lmsstudent.EdFiStudentId = edfistudent.Id
 	INNER JOIN edfi.Section edfisection
-		ON edfisection.SectionIdentifier = lmsxassignment.SectionIdentifier
+		ON edfisection.SectionIdentifier = edfisectionassociation.SectionIdentifier
 	INNER JOIN lms.LMSSection lmssection
 		ON lmssection.EdFiSectionId = edfisection.Id
 	LEFT JOIN lms.AssignmentSubmission lmssubmission
-		ON lmssubmission.AssignmentIdentifier = lmsxassignment.AssignmentIdentifier
+		ON lmssubmission.AssignmentIdentifier = lmsassignment.AssignmentIdentifier
 			AND lmssubmission.LMSUserIdentifier = lmsstudent.LMSUserIdentifier
 	CROSS APPLY (
         SELECT
             submsisionstatusdescriptor.DescriptorId
         FROM
             edfi.Descriptor submsisionstatusdescriptor
 		WHERE
-            submsisionstatusdescriptor.Namespace = 'uri://ed-fi.org/edfilms/SubmissionStatusDescriptor/' + @SourceSystem
+            submsisionstatusdescriptor.Namespace = 'uri://ed-fi.org/edfilms/SubmissionStatusDescriptor/' + 'Schoology'
 		AND
             submsisionstatusdescriptor.CodeValue = 'missing'
     ) as submsisionstatusdescriptor
 	WHERE lmssubmission.LMSUserIdentifier IS NULL
 	AND lmsxassignment.DueDateTime < GETDATE()
+	AND @SourceSystem = 'Schoology'
```

Comment:
For better clarity when reading the script from top to bottom, what do you think about changing this around so that we have

```
IF @SourceSystem = 'Schoology'
BEGIN
    -- do stuff
END
```

instead of having this clause at the end of the statement?